.travis.yml

```diff
@@ -0,0 +1,19 @@
+sudo: false
+language: node_js
+cache:
+  yarn: true
+notifications:
+  email: false
+node_js:
+  - '12'
+script:
+  - yarn run lint
+  - yarn run lint-docs
+  # the test suite doesn't quite work on Travis because of the way Travis
+  # mucks with repositories
```

Comment:
One final comment, can you link to a fuller description of how Travis interferes (if this is documented somewhere)?